### PR TITLE
refactor(LINGUIST-410): Change item descriptions

### PR DESCRIPTION
**Pull request by KardelenCeren**
Changed the item descriptions. Open for feedback :)

### CODE REVIEW

File: src/main/java/app/linguistai/bmvp/consts/StoreConsts.java [hunk 1]
```diff
@@ -7,8 +7,8 @@ public class StoreConsts {
     public static final String[] QUIZ_TYPES = {TYPE_DOUBLE_ANSWER, TYPE_ELIMINATE_WRONG_ANSWER};
 
     // Descriptions
-    public static final String DESCRIPTION_DOUBLE_ANSWER = "Unlock the ability to answer a question twice for better practice.";
-    public static final String DESCRIPTION_ELIMINATE_WRONG_ANSWER = "Unlock the ability to eliminate two wrong answers during a quiz.";
+    public static final String DESCRIPTION_DOUBLE_ANSWER = "Consume this item to answer a quiz question twice for better practice.";
```
Comment (can-aslan) [inline src/main/java/app/linguistai/bmvp/consts/StoreConsts.java:10]: `Consume this item to have two attempts for a quiz question.` might be more user friendly

**Review by can-aslan — DISMISSED** [review]
LGTM, just left a small suggestion if you wish to change it

File: src/main/java/app/linguistai/bmvp/consts/StoreConsts.java (hunk 1 above)
Comment (KardelenCeren) [reply src/main/java/app/linguistai/bmvp/consts/StoreConsts.java]: Very true, changed it!

**Review by tolgaozgun — APPROVED** [review]
LGTM

**Pull request merged**